structuredefinitions/Extension-UKCore-SubscriptionNotificationPayloadSize.xml

```diff
@@ -0,0 +1,55 @@
+  <url value="http://hl7.org/fhir/5.0/StructureDefinition/extension-Subscription.content" />
+  <version value="0.0.1" />
+  <name value="ExtensionUKCoreSubscriptionNotificationPayloadSize" />
+  <title value="Extension UK Core Subscription Notification Payload Size" />
+  <status value="draft" />
+  <date value="2025-03-10" />
+  <publisher value="HL7 UK" />
+  <contact>
+    <name value="HL7 UK" />
+    <telecom>
+      <system value="email" />
+      <value value="ukcore@hl7.org.uk" />
+      <use value="work" />
+      <rank value="1" />
+    </telecom>
+  </contact>
+  <description value="An extension to replicate the changes within R5 to support the indication of how much of the resource content to deliver in a notification payload. This is a backport of R5 Subscription.content element." />
+  <purpose value="This extension is a Multicast Network Service (MNS) use case, require to indicate how much of the resource content to deliver in a notification payload (specifying decision to received either a hydrated or full resource)." />
+  <copyright value="Copyright © 2021+ HL7 UK Licensed under the Apache License, Version 2.0 (the &quot;License&quot;); you may not use this file except in compliance with the License. You may obtain a copy of the License at  http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an &quot;AS IS&quot; BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License. HL7® FHIR® standard Copyright © 2011+ HL7 The HL7® FHIR® standard is used under the FHIR license. You may obtain a copy of the FHIR license at  https://www.hl7.org/fhir/license.html." />
+  <fhirVersion value="4.0.1" />
+  <kind value="complex-type" />
+  <abstract value="false" />
+  <context>
+    <type value="element" />
+    <expression value="Subscription" />
+  </context>
+  <type value="Extension" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Extension" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Extension">
+      <path value="Extension" />
+      <mustSupport value="true" />
+    </element>
+    <element id="Extension.url">
+      <path value="Extension.url" />
+      <fixedUri value="http://hl7.org/fhir/5.0/StructureDefinition/extension-Subscription.content" />
+    </element>
+    <element id="Extension.value[x]">
+      <path value="Extension.value[x]" />
+      <short value="Yes or No" />
+      <type>
+        <code value="code" />
+      </type>
+      <binding>
+        <strength value="required" />
+        <description value="An indication of choice to options of a subscription message (whether to subscribe to hydrated resource or full resource)." />
+        <valueSet value="http://terminology.hl7.org/ValueSet/v2-0136" />
```

Comment:
Can't find this valueset, shouldn't the valueset be a backport of https://hl7.org/fhir/R5/valueset-subscription-payload-content.html?

Reply:
My understanding of the description on the [JIRA ticket](https://nhsd-jira.digital.nhs.uk/browse/IOPS-3003) suggests a Yes or No situation. Hence, the reason I mapped to valueset yes or no indicator. However, on the cross version issue, ValueSet are independent of FHIR versions. The correct Canonical-URL to use will be: http://hl7.org/fhir/ValueSet/subscription-payload-content